lib/dmv_facility.rb

```diff
@@ -0,0 +1,28 @@
+class DmvFacility
+  def create_facilities(databases)
+    datum = databases.map do |data|
+      Facility.new({
+        name: access_name(data),
```

Comment:
I love that you used helper methods to extract and parse this data! Great solution